backend/open_webui/config.py

```diff
@@ -1840,10 +1840,10 @@ class BannerModel(BaseModel):
 QDRANT_URI = os.environ.get("QDRANT_URI", None)
 QDRANT_API_KEY = os.environ.get("QDRANT_API_KEY", None)
 QDRANT_ON_DISK = os.environ.get("QDRANT_ON_DISK", "false").lower() == "true"
-QDRANT_PREFER_GRPC = os.environ.get("QDRANT_PREFER_GRPC", "False").lower() == "true"
+QDRANT_PREFER_GRPC = os.environ.get("QDRANT_PREFER_GRPC", "false").lower() == "true"
 QDRANT_GRPC_PORT = int(os.environ.get("QDRANT_GRPC_PORT", "6334"))
 ENABLE_QDRANT_MULTITENANCY_MODE = (
-    os.environ.get("ENABLE_QDRANT_MULTITENANCY_MODE", "false").lower() == "true"
+    os.environ.get("ENABLE_QDRANT_MULTITENANCY_MODE", "true").lower() == "true"
```

Comment:
Using a collection per user/document is an anti-pattern in Qdrant. It significantly impacts perf.
Multi-tenancy based on payload is recommended. Hence, it should be the default.

I'll also advocate depracating [qdrant.py](https://github.com/open-webui/open-webui/blob/59ba21bdf8eb791a412db869a13ff76c6135b651/backend/open_webui/retrieval/vector/dbs/qdrant.py#L30) in favour of [qdrant_multitenancy.py](https://github.com/Anush008/open-webui/blob/17debaa6de1537ff336d0c16fed64e2c08a3c025/backend/open_webui/retrieval/vector/dbs/qdrant_multitenancy.py) completely.